src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -205,6 +206,23 @@ protected set
             }
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TransformedByLayout
+        {
+            get
+            {
+                return transformedByLayout;
+            }
+            set
+            {
+                transformedByLayout = value;
+                if(Layout != null && Layout.SetPositionByLayout != value)
+                {
+                    Layout.SetPositionByLayout = value;
```

Comment:
I think `Layout.RequestLayout()` needs to be called for re-layout when `SetPositionByLayout` value is changed.

Reply:
Thank you it is needed.